UIKit_DZ/UIKit_DZ/FeedScreen/PostCell/PostBody.swift

```diff
@@ -0,0 +1,182 @@
+// PostBody.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Основная часть ячейки поста c изображениями, функциональными элементами и счетчиком лайков
+final class PostBody: UIView {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let amountOfLikesText = "Нравится: "
+    }
+
+    // MARK: - Visual Components
+
+    private let likeImageView: UIImageView = {
+        let view = UIImageView()
+        view.contentMode = .scaleAspectFit
+        view.image = .likeIcon.withTintColor(.accent)
+        return view
+    }()
+
+    private let commentImageView: UIImageView = {
+        let view = UIImageView()
+        view.contentMode = .scaleAspectFit
+        view.image = .messageIcon.withTintColor(.accent)
+        return view
+    }()
+
+    private let shareImageView: UIImageView = {
+        let view = UIImageView()
+        view.contentMode = .scaleAspectFit
+        view.image = .shareIcon.withTintColor(.accent)
+        return view
+    }()
+
+    private let imageScrollPageControl: UIPageControl = {
+        let control = UIPageControl()
+        control.currentPage = 0
+        control.hidesForSinglePage = true
+        control.isUserInteractionEnabled = false
+        control.pageIndicatorTintColor = .opaqueSeparator
+        control.currentPageIndicatorTintColor = .accent
+        return control
+    }()
+
+    private let bookMarkImageView: UIImageView = {
+        let view = UIImageView()
+        view.contentMode = .scaleAspectFit
+        view.image = .bookmarkIcon.withTintColor(.accent)
+        return view
+    }()
+
+    private let numberOfLikesLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .accent
+        label.font = .verdanaBold?.withSize(10)
+        label.textAlignment = .left
+        label.text = Constants.amountOfLikesText
+        return label
+    }()
+
+    private lazy var imageScrollView: UIScrollView = {
+        let scroll = UIScrollView()
+        scroll.isPagingEnabled = true
+        scroll.bounces = false
+        scroll.showsVerticalScrollIndicator = false
+        scroll.showsHorizontalScrollIndicator = false
+        scroll.delegate = self
+        return scroll
+    }()
+
+    // MARK: - Private Properties
+
+    private var imageScrollImageViews: [UIImageView] = []
```

Comment:
```suggestion
    private var imageScrollImageViews: [UIImageView] = []
```
ui лучше так не хранить